scipy-2023-data-grammar-poster/poster.rmd

```diff
@@ -0,0 +1,125 @@
+---
+title: Cross-language Data Grammar for Single-cell Research Feature Engineering
+author:
+    - name: Dave Bunten
+      affil: 1
+      orcid: 0000-0001-6041-3665
+      main: true
+    - name: Faisal Alquaddoomi
+      affil: 1
+      orcid: 0000-0003-4297-8747
+      main: true
+    - name: Gregory P. Way
+      affil: 1
+      orcid: 0000-0002-0503-9348
+      main: true
+affiliation:
+  - num: 1
+    address: Department of Biomedical Informatics, University of Colorado Anschutz
+main_findings:
```

Comment:
Is it worth including `parcel` here? Or we shouldn't b/c it is not in the abstract? (it was prefect before)?

Reply:
Great point, thanks! I'll add this in here and update the diagram.